infra/base-images/base-clang/checkout_build_install_llvm.sh

```diff
@@ -24,6 +24,10 @@ NPROC=$(expr $(nproc) / 2)
 LLVM_DEP_PACKAGES="build-essential make cmake ninja-build git python3 python3-distutils g++-multilib binutils-dev zlib1g-dev"
 apt-get update && apt-get install -y $LLVM_DEP_PACKAGES --no-install-recommends
 
+if [ -n "$INTROSPECTOR_PATCHES" ]; then
+  apt-get install -y texinfo bison flex
```

Comment:
Can we delete these after the install?

Reply:
will look into it.

Reply:
looks removing does not break it.

Reply:
Can you set this similarly to LLVM_DEP_PACKAGES, so these are defined in one place.

i.e.

```
INTROSPECTOR_DEP_PACKAGES="texinfo bison flex"
```

Reply:
Done.